src/amo/containers/Home.js

```diff
@@ -6,6 +7,22 @@ import translate from 'core/i18n/translate';
 
 import 'amo/css/Home.scss';
 
+const CategoryLink = ({ name, slug, text, type }) => (
```

Comment:
I'd prefer components complex enough to have proptypes to just be in their own files (I just recently did this in an upcoming PR and find it's easier to grok things when there is a single component per file). Plus I dislike declaring proptypes like that.